packages/react-server-website/components/Footer.js

```diff
@@ -4,9 +4,16 @@ export default class Footer extends React.Component {
 	render() {
 		return (
 			<footer>
-				<div>
-					&copy;2016 <a href="https://www.redfin.com/">Redfin</a>.
-				</div>
+				<p>
+					<small>
+						<a href="https://www.redfin.com/">Redfin</a> open-source project. &copy;2016 Redfin.
+					</small>
+				</p>
+				<p>
+					<small>
+						<a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License 2.0</a> Talk with us on <a href="https://react-server.slack.com/">Slack</a>.
```

Comment:
This slack link will probably not be what we want.  We're going to have a [slackin](https://github.com/rauchg/slackin) gateway.